File: 1. 23.08/Java/7.java
import java.util.*;

class Main {
	public static void main(String[] args)
	{	
		Scanner sc= new Scanner(System.in);
		System.out.print("Enter 1 number: ");
		int a = sc.nextInt();
		System.out.print("Enter 2 number: ");
		int b = sc.nextInt();
	
		int temp = a;
		a = b;
		b = temp;
		System.out.printf("Num 1 is: %d\n",a);
		System.out.printf("Num 2 is: %d\n",b);
	}
}

File: 1. 23.08/C++/7.cpp
#include <iostream>

void swap(int &a, int &b)
{
	int temp = a;
	a = b;
	b = temp;
}

int main()
{
	int a,b;
	std::cout<<"Enter A:"<<std::endl;
	std::cin>>a;
	std::cout<<"Enter B:"<<std::endl;
	std::cin>>b;
	swap(a,b);
	std::cout<<"A is: " << a << "\nB is: " << b << std::endl;
	return 0;
}
